app/models/katello/content_view_history.rb

```diff
@@ -66,5 +66,13 @@ def humanized_status
         _("Success")
       end
     end
+
+    def description
+      if self.task.try(:label) == "Actions::Katello::ContentView::Publish"
+        content_view_version.description
```

Comment:
Out of curiosity, is there a historical, technical or use case reason that we use version.description for Publish, instead of using history.notes?